scripts/hermes-binary.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+[ -z "$HERMES_PATH" ] && export HERMES_PATH="$REACT_NATIVE_DIR/../hermes-engine-darwin"
+
+export HERMES_CLI = "$HERMES_PATH/destroot/bin/hermesc"
+
+if [[ $USE_HERMES == true && ! -d "$HERMES_PATH" ]]; then
+  echo "error: Can't find Hermes executable - directory `$HERMES_PATH` doesn't exist. " \
+       "If you have a non-standard project structure, select your project in Xcode, find " \
+       "'Build Phases' - 'Bundle React Native code and images' and set `HERMES_PATH` to an " \
+       "absolute path to a `hermes-engine-darwin` package inside your `node_modules`" >&2
```

Comment:
Probably not absolute, as that’s not portable between machines. Does the code need updating to support relative paths?

Reply:
Relative path from project root would work. Good catch!

Reply:
I think users should set it absolute to avoid issues with PWD, e.g. by using the `SRCROOT` variable, which is already incorporated inside `react-native-xcode.sh`:

```
"$SRCROOT/../../scripts/react-native-xcode.sh"
```

Reply:
Hmm, I guess this is ambiguous language then, because I would consider the example you gave to be a relative one. I.e. before expansion it is relative to `$SRCROOT`.

Reply:
Would you be able to help me here with a better wording? That would be greatly appreciated!

Reply:
How about an example?

> If you have a non-standard project structure, locate the 'Bundle React Native code and images' build phase in your Xcode project and set the `HERMES_PATH` variable to the location of the `hermes-engine-darwin` package. For example: `$SRCROOT/../../node_modules/hermes-engine-darwin`

Reply:
Awesome!